src/Mutation.php

```diff
@@ -0,0 +1,186 @@
+<?php
+
+/**
+ * Mutation to apply to a source file
+ *
+ * @category   Humbug
+ * @package    Humbug
+ * @copyright  Copyright (c) 2015 Pádraic Brady (http://blog.astrumfutura.com)
+ * @license    https://github.com/padraic/humbug/blob/master/LICENSE New BSD License
+ * @author     Thibaud Fabre
+ */
+namespace Humbug;
+
+class Mutation
+{
+    /**
+     * @var string
+     */
+    private $file;
+
+    /**
+     * @var int
+     */
+    private $line;
+
+    /**
+     * @var string
+     */
+    private $class;
+
+    /**
+     * @var string
+     */
+    private $method;
+
+    /**
+     * @var int
+     */
+    private $index;
+
+    /**
+     * @var string
+     */
+    private $mutator;
+
+    /**
+     * @param string $file
+     * @param int $line
+     * @param string $class
+     * @param string $method
+     * @param int $index
+     * @param string $mutator
+     */
+    public function __construct($file, $line, $class, $method, $index, $mutator)
+    {
+        $this->file = $file;
+        $this->line = $line;
+        $this->class = $class;
+        $this->method = $method;
+        $this->index = $index;
+        $this->mutator = $mutator;
+    }
+
+    /**
+     * (PHP 5 &gt;= 5.0.0)<br/>
+     * Whether a offset exists
+     *
+     * @link http://php.net/manual/en/arrayaccess.offsetexists.php
+     *
+     * @param mixed $offset <p>
+     * An offset to check for.
+     * </p>
+     *
+     * @return boolean true on success or false on failure.
+     * </p>
+     * <p>
+     * The return value will be casted to boolean if non-boolean was returned.
+     */
+    public function offsetExists($offset)
```

Comment:
You removed \ArrayAccess so what is it should be removed also - like other methods from \ArrayAccess interface